README.md

```diff
@@ -1,29 +1,40 @@
 # CoAP CLI
-Simple CoAP cli client written in Go.
 
+Simple CoAP cli client written in Go.
 
 ## Usage
-Pre-built binary can be found here: https://github.com/mainflux/coap-cli/releases/tag/v0.3.3.
-When running, please provide following format:
-`coap-cli` followed by method code (`get`, `put`, `post`, `delete`) and CoAP URL. After that, you can pass following flags:
-
-| Flag | Description                                    | Default value    |
-| ---- | ---------------------------------------------- | ---------------- |
-| o    | observe   option - only valid with GET request | false            |
-| auth | auth option sent as URI Query                  | ""               |
-| h    | host                                           | "localhost"      |
-| p    | port                                           | "5683"           |
-| d    | data to be sent in POST or PUT                 | ""               |
-| cf   | content format                                 | 50 (JSON format) |
+
+coap-cli [`command`]
+
+#### Available Commands:
+
+- `completion` Generate the autocompletion script for the specified shell
+- `delete` Perform a DELETE request on a COAP resource
+- `get` Perform a GET request on a COAP resource
+- `help` Help about any command
+- `post` Perform a POST request on a COAP resource
+- `put` Perform a PUT request on a COAP resource
+
+#### Flags:
+
+- `-a`, `--auth` string Auth (default "")
+- `-c`, `--content-format` int Content format (default 50)
+- `-h`, `--help` help for coap-cli
+- `-H`, `--host` string Host (default "localhost")
+- `-p`, `--port` string Port (default "5683")
+
```

Comment:
Please add `-o`, with note like `it works for only get`

Reply:
Observe option is one of the main features of CoAP, please don't forget this.